oci/spec_opts_unix.go

```diff
@@ -368,7 +369,11 @@ func WithUserID(uid uint32) SpecOpts {
 				return err
 			}
 			s.Process.User.UID, s.Process.User.GID = uuid, ugid
-			return nil
+			sgids, err := GetAdditionalGIDs(s.Root.Path, string(uid))
```

Comment:
Modify the existing `getUIDGIDFromPath` function to return supplementary groups as well instead

Reply:
Done.